### PR TITLE
[bitnami/minio] add commonLabels to template.metadata

### PR DESCRIPTION
**Pull request by clearly-tyler-whitney**
### Description of the change

Adds `commonLabels` to standalone MinIO `deployment.spec.template.spec.metadata` in the style of existing Bitnami charts.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @clearly-tyler-whitney,

Could you please bump the version of the chart to `11.10.9`? Version 11.10.8 was already released.

In addition, it would be great if you could apply the same change to the statefulset for 'distributed' mode.

File: bitnami/minio/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 11.10.7
+version: 11.10.8
```
[the remark this answers is not in the document]
Comment (clearly-tyler-whitney) [reply bitnami/minio/Chart.yaml]: Bumped to 11.10.10

**Comment by clearly-tyler-whitney** [comment]
> Hi @clearly-tyler-whitney,
> 
> Could you please bump the version of the chart to `11.10.9`? Version 11.10.8 was already released.
> 
> In addition, it would be great if you could apply the same change to the statefulset for 'distributed' mode.

Gladly. Title updated.

**Comment by migruiz4** [comment]
Hi @clearly-tyler-whitney,

The commit https://github.com/bitnami/charts/pull/12962/commits/f4b55cb275715a22f571822aa575069065e79cd6 wasn't signed and is causing the DCO to fail. Could you please amend this issue?

**Comment by clearly-tyler-whitney** [comment]
> Hi @clearly-tyler-whitney,
> 
> The commit [f4b55cb](https://github.com/bitnami/charts/commit/f4b55cb275715a22f571822aa575069065e79cd6) wasn't signed and is causing the DCO to fail. Could you please amend this issue?

Hi @migruiz4, the DCO issue has been addressed. Apologies for the hassle.

**Review by migruiz4 — APPROVED** [review]
Thank you very much! Looks good to me!

**Pull request merged**